content/en/platform/corda/1.2/cenm/deployment-kubernetes.md

```diff
@@ -72,6 +72,11 @@ Each CENM service has its own dedicated folder with more detailed documentation.
 The charts are currently developed and tested against
 [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-gb/services/kubernetes-service/).
 
+### VM sizing guidelines
+
+Recommended production specification for CENM components running on separate VMs:
+* 2 CPU Core, 8 GB Memory
```

Comment:
```suggestion
* 2 CPU cores, 8GB memory
```

Repeat for other sections.

Reply:
In Corda Ent Docs we have "Core" not "cores": https://docs.r3.com/en/platform/corda/4.10/enterprise/node/setup/host-prereq.html#vm-sizing-guidelines but I see online that "cores" is probably more correct so changing to "cores".
Leaving "8 GB" as per MS style guide: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms